src/components/LinkSearch.js

```diff
@@ -0,0 +1,11 @@
+import "./styles/LinkSearch.css";
+
+function LinkSearch() {
+  return (
+    <div className="LinkSearch">
+      <input placeholder="링크를 검색해 보세요."></input>
```

Comment:
하위 요소를 가지고 있지 않다면 self-closing 하시면 됩니다.

```suggestion
      <input placeholder="링크를 검색해 보세요." />
```